Arrays/135Candy.py

```
@@ -0,0 +1,27 @@
+from collections import defaultdict
+class Solution:
+      def candy(self, ratings: list[int]) -> int:
+          result=[1]*len(ratings)
+          for i in range(1,len(ratings)):
+              if ratings[i-1]<ratings[i]:
+                 result[i]+=result[i-1]
+          for i in range(len(ratings)-1,0,-1):
+              if ratings[i]<ratings[i-1]:
+               
+                 if result[i]>=result[i-1]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)